designs/2022-docs-information-architecture-update/README.md

```diff
@@ -0,0 +1,252 @@
+- [MOVE & RENAME PAGE] Propose a Rule Change
+  - Current page: [Proposing a Rule Change](https://eslint.org/docs/latest/developer-guide/contributing/#:~:text=Proposing%20a-,Rule%20Change,-Want%20to%20make)
+- [MOVE & RENAME PAGE] Request a Change
+  - Current page: [Change Requests](https://eslint.org/docs/latest/developer-guide/contributing/changes)
+- [MOVE & RENAME PAGE] Work on Issues
+  - Current page: [Working on Issues](https://eslint.org/docs/latest/developer-guide/contributing/working-on-issues)
+- [NEW PAGE] Report Security Vulnerabilities
+  - Tales current content from the [Reporting a Security Vulnerability section on the Contributing page](https://eslint.org/docs/latest/developer-guide/contributing/#reporting-a-security-vulnerability). Worth breaking out into its own page for discoverability and SEO for an important subject.
+  - NOTE: Should be added, but not part of the initial IA project. Probably can be done as a one-off task or group in with one of the project phases.
+- [MOVE & RENAME PAGE] Submit a Pull Request
+  - Current page: [Pull Requests](https://eslint.org/docs/latest/developer-guide/contributing/pull-requests)
+
+## Drawbacks
+
+Potential drawbacks of this documentation update include:
+
+- Breaking user experience from the current documentation
+- Users potentially do not find the new information architecture more clear
+- Opportunity cost. Time spent on the information architecture updates could be spent on more impactful documentation tasks.
+
+<!--
+    Why should we *not* do this? Consider why adding this into ESLint
+    might not benefit the project or the community. Attempt to think
+    about any opposing viewpoints that reviewers might bring up.
+
+    Any change has potential downsides, including increased maintenance
+    burden, incompatibility with other tools, breaking existing user
+    experience, etc. Try to identify as many potential problems with
+    implementing this RFC as possible.
+-->
+
+## Backwards Compatibility Analysis
+
+HTTP redirects from existing content URL paths to the new content URL paths must be included as part of this project.
+
+<!--
+    How does this change affect existing ESLint users? Will any behavior
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+## Alternatives
+
+- Do not change the IA. Only update documentation within the existing IA as part of the [documentation update project](https://github.com/eslint/eslint/issues/16365).
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+## Open Questions
+
+1. Is there a way to automatically change all the internal links that'll be broken by these updates?
+1. Can you create a side navigation folder that is not also a page?
```

Comment:
i mean a sidebar container that displays a name and has child elements, but isn't a page with content. 

in Docusaurus, this is called a [container](https://docusaurus.io/docs/sidebar/items#sidebar-item-category)

Reply:
Ah gotcha. The answer is yes, those don’t need to have landing pages. We have them currently because there are existing links out there that point to those pages but we don’t actually link them from the sidebar.

Reply:
Oops sorry, misunderstood this.

So we have the top level items like User Guide that aren’t linked to anything. We don’t have the ability to do the same anywhere else in the sidebar, so we always need a landing page for anything other than that top level. This was a design decision to make it clear what was clickable and what wasn’t.

Reply:
ok thanks for clarifying this. will keep this consideration in mind for the IA changes. right now going thru what's already written so see if any updates necessary. will add if so.